tools/editorconfig/.editorconfig

```diff
@@ -10,6 +10,9 @@ trim_trailing_whitespace = true
 [*.{yml,twig,php}]
 indent_size = 4
 
+[*.{js,css}]
```

Comment:
Do we have another asset extension like `.less`, `.scss`, `.coffee` on sonata projects?